test/node/transport-w3c-ws.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const test = require('tap');
+global.WebSocket = require('websocket').w3cwebsocket;
+const w3c = require('../../lib/ws-browser');
+
+const jstp = require('../..');
+
+const app = require('../fixtures/application');
+const application = new jstp.Application(app.name, app.interfaces);
+
+const interfaces = Object.keys(app.interfaces);
+
+let server;
+let connection;
+
+test.beforeEach((done) => {
+  server = jstp.ws.createServer([application]);
+  server.listen(0, done);
+});
+
+test.afterEach((done) => {
+  if (connection) {
+    connection.close();
+    connection = null;
+  }
+  server.close();
+  done();
+});
+
+test.test('W3C WebSocket connection must connect to server', (test) => {
```

Comment:
Maybe smth like 'Must connect to the server via W3C WebSocket' and 'Must connect to the server and inspect via W3C WebSocket', here as well as in other places?